src/cpu/amd/pi/00730F01/Kconfig

```diff
@@ -62,4 +62,10 @@ config HIGH_SCRATCH_MEMORY_SIZE
 	# Assume the maximum size of stack as (0xA0000 - 0x30000 + 0x1000)
 	default 0xA1000
 
+config CPU_AMD_FAM16H_MICROCODE
```

Comment:
@miczyg1 please change that and make it option to add microcode binary by providing path to it as `bootorder` file and other SeaBIOS configuration files. Also please consider sending this series to mailing list for comments.